File: Codeforces/C++/problemset/Word.cpp
// Problem Link: https://codeforces.com/problemset/problem/59/A

#include <iostream>
#include <string>
using namespace std;
int main()
{
    string s;
    cin >> s;
    long long u = 0,l = 0;
    for(long long i = 0; i < s.size(); i++)
    {
        if(isupper(s[i])) u++;
        if(islower(s[i])) l++;
    }
    if(l >= u)
    {
        for(long long i = 0; i < s.size(); i++)
        {
            s[i] = tolower(s[i]);
        }
    }
    else
    {
        for(long long i = 0; i < s.size(); i++)
        {
            s[i] = toupper(s[i]);
        }
    }
    cout << s;

    return 0;
}

File: src/main/kotlin/Word.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/59/A */

fun main() {
    val word = readln()
    val upper = word.count { it.isUpperCase() }
    val lower = word.length - upper

    print(if (upper > lower) word.uppercase() else word.lowercase())
}